mypy/modulefinder.py

```diff
@@ -472,15 +466,53 @@ def make_abspath(path: str, root: str) -> str:
         site_packages = ast.literal_eval(
             subprocess.check_output([python_executable, sitepkgs.__file__],
             stderr=subprocess.PIPE).decode())
-    egg_dirs = []
+    return expand_site_packages(site_packages)
+
+
+def expand_site_packages(site_packages: List[str]) -> Tuple[List[str], List[str]]:
+    """Expands .pth imports in site-packages directories"""
+    egg_dirs = []  # type: List[str]
     for dir in site_packages:
-        pth = os.path.join(dir, 'easy-install.pth')
-        if os.path.isfile(pth):
-            with open(pth) as f:
-                egg_dirs.extend([make_abspath(d.rstrip(), dir) for d in f.readlines()])
+        if not os.path.exists(dir):
+            continue
+        pth_filenames = sorted(name for name in os.listdir(dir) if name.endswith(".pth"))
+        for pth_filename in pth_filenames:
+            egg_dirs.extend(_parse_pth_file(dir, pth_filename))
+
     return egg_dirs, site_packages
 
 
+def _parse_pth_file(dir: str, pth_filename: str) -> Iterator[str]:
+    """
+    Mimics a subset of .pth import hook from Lib/site.py
+    cref https://github.com/python/cpython/blob/3.5/Lib/site.py#L146-L185
```

Comment:
"cref" is an obscure abbrev -- I'd just use "See". Also, why reference the 3.5 branch? It's so old!

Reply:
I picked the 3.5 branch b/c that's the min version supported by mypy.